packages/github-actions/actions/get-plugin-releases/src/get-plugin-releases.js

```diff
@@ -53,12 +53,14 @@ function parsePluginVersions( releases = {} ) {
 	const output = [];
 
 	if ( slug !== 'wordpress' ) {
+		const latest = releases.version;
 		const versions = Object.keys( releases.versions )
 			.filter(
 				( version ) =>
 					version !== 'trunk' &&
 					version !== 'other' &&
-					! version.includes( 'beta' )
+					! version.includes( 'beta' ) &&
+					semverCompare( latest, version ) <= 0
```

Comment:
As far as I understand this will work if we aren't interested in RC versions. But if the stable version is at `8.7.0` and a `8.8.0-rc.1` is available it will now be filtered out.

Reply:
Thanks @mikkamp for catching that. Added in https://github.com/woocommerce/grow/pull/102/commits/a9ddff799f70725c76fa921fd509fd239b4ab00f